doc/development_guide/api/pylint.rst

```diff
@@ -0,0 +1,76 @@
+=======
+ Pylint
+=======
+
+As you would launch the command line
+------------------------------------
+
+You can use the ``run_pylint`` function, which is the same function
+called by the command line. You can either patch ``sys.argv``:
+
+.. sourcecode:: python
+
+    from pylint import run_pylint
+
+    sys.argv = ["pylint", "--disable=line-too-long", "myfile.py"]
+    run_pylint()
+
+
+Or supply arguments yourself:
+
+.. sourcecode:: python
+
+    from pylint import run_pylint
+
+    run_pylint(argv=["--disable=line-too-long", "myfile.py"])
```

Comment:
Considering:
```suggestion
As you would launch the command line
------------------------------------

You can use the ``run_pylint`` function, which is the same function
called by the command line.

.. sourcecode:: python

    from pylint import run_pylint

    run_pylint(argv=["--disable=line-too-long", "myfile.py"])
```

> There should be one obvious way to do it

Reply:
Yeah I like the suggestion better although I would keep `Or supply arguments yourself:` at the end.

Reply:
[e87b2ce](https://github.com/PyCQA/pylint/pull/6658/commits/e87b2ce0deab9adaea0fd493c03e0a4d5a327774) ? Not sure if it is what you meant :)